eos-core-depends

```diff
@@ -58,3 +58,94 @@ unoconv
 unrar
 whiptail
 xdg-app
+
+
+# moved here from os-depends due to installer
+chromium-browser
+cups
+eos-social
+eos-speedwagon
+eos-tech-support
+eos-tutorial
+eos-updater
+evince
+fake-hwclock
+file-roller
+fonts-arabeyes
+fonts-arphic-ukai
+fonts-arphic-uming
+fonts-cabinsketch
+fonts-crosextra-caladea
+fonts-crosextra-carlito
+fonts-dejavu
+fonts-dosis
+fonts-droid
+fonts-dustin
+fonts-farsiweb
+fonts-gargi
+fonts-indic
+fonts-kacst
+fonts-khmeros
+fonts-knda
+fonts-lato
+fonts-nanum
+fonts-nanum-coding
+fonts-ocr-a
+fonts-paktype
+fonts-quattrocento
+fonts-roboto
+fonts-sil-andika
+fonts-sil-padauk
+fonts-thai-tlwg
+fonts-tlwg-waree
+fonts-unfonts-core
+fonts-uralic
+fonts-vlgothic
+foomatic-db-compressed-ppds
+fprintd
+gedit
+gnome-calculator
+gnome-clocks
+gnome-control-center
+gnome-orca
+gnome-screenshot
+gnome-software
+gnome-sushi
+gnome-system-monitor
+gnome-terminal
+grilo-plugins-0.2
+gvfs-bin
+ibus-anthy
+ibus-cangjie
+ibus-chewing
+ibus-hangul
+ibus-input-pad
+ibus-libpinyin
+ibus-libthai
+ibus-libzhuyin
+ibus-m17n
+ibus-table-wubi
+ibus-tegaki
+ibus-unikey
```

Comment:
Input methods might not be quite as important, as the user is probably not entering anything in non-English characters -- just clicking with the mouse, I think.